src/pages/index.js

```diff
@@ -85,30 +96,47 @@ profilePostButton.addEventListener("click", (evt) => {
   disableButton(modalButtonPost, settings);
```

Comment:
You need to **disable** the button only in a `then` block after a **successful response** to be able to click it again if there is a server error.